recipes/bioconductor-codex/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: bioconductor-codex
+  version: 1.2.0
+
+source:
+  fn: CODEX_1.2.0.tar.gz
+  url: https://www.bioconductor.org/packages/release/bioc/src/contrib/CODEX_1.2.0.tar.gz
```

Comment:
Did you use the bioconductor script in the scripts directory? I thought it would do that automatically...